lit_nlp/client/modules/attention_module.ts

```diff
@@ -204,6 +204,16 @@ export class AttentionModule extends LitModule {
     // clang-format on
   }
 
+  private getTokOffsets(tokWidths: number[], spaceWidth: number): number[] {
```

Comment:
Add to utils.ts? Might be useful for other visualizations

Reply:
I wasn't sure if this is broadly useful, so I put it here. I can move it to utils.ts, if you would like.

Reply:
Yes, that would be great, thanks-- token spacing comes up pretty frequently

Reply:
Done